src/EmployeesAdministrationApp.java

```diff
@@ -12,7 +14,7 @@
 import org.json.simple.parser.JSONParser;
 
 public class EmployeesAdministrationApp {
-	public static JSONArray listOfEmployees = new JSONArray();
+	public static List<HashMap<String, Object>> listOfEmployees = new ArrayList<HashMap<String, Object>>();
```

Comment:
this should be HashMap<String, String>, as we discussed, where the key should be the ID and the value should be a string containing all the employee information

Reply:
I changed the list of employees from List to HashMap